runtimes/nodejs/actions/send-message.js

```diff
@@ -440,3 +464,45 @@ function isEmpty(obj) {
   }
   return true;
 }
+
+/**
+* Helper function used to authenticate credentials bound to package using wsk service bind
+*
+* @param {Object} theParams - parameters sent to service
+*/
+function getParams(theParams) {
+  const service = 'imfpush';
+  if (Object.keys(theParams).length === 0) {
+    return theParams;
+  }
+  let bxCreds = {};
+  // Code that checks parameters bound using service bind
+  if (theParams.__bx_creds && theParams.__bx_creds[service]) {
+    bxCreds = theParams.__bx_creds[service];
+  }
+  const allParams = Object.assign({}, bxCreds, theParams);
+  delete allParams.__bx_creds;
+  return allParams;
+}
+
+function handleAuth(triggerData) {
```

Comment:
I would rename this parameter since it is not related to triggers

Reply:
done, thanks!